easybuild/easyconfigs/x/XCrySDen/XCrySDen-1.6.2-intel-2021b.eb

```diff
@@ -0,0 +1,40 @@
+# Contribution from IT4Innovations National Supercomputing Center, Czech Republic
+# Jakub Kropacek, 2022
+
+name = 'XCrySDen'
+version = '1.6.2'
+
+homepage = "http://www.xcrysden.org/"
+docurls = "http://www.xcrysden.org/Documentation.html"
+description = """
+XCrySDen is a crystalline and molecular structure visualisation program aiming at display of isosurfaces
+and contours, which can be superimposed on crystalline structures and interactively rotated and manipulated.
+It also possesses some tools for analysis of properties in reciprocal space such as interactive selection
+of k-paths in the Brillouin zone for the band-structure plots, and visualisation of Fermi surfaces.
+"""
+
+toolchain = {'name': 'intel', 'version': '2021b'}
```

Comment:
I don't see this software using MPI or MKL, so could be dropped to `intel-compilers`

Reply:
OK, done, no strong opinions on this.

Reply:
@verdurin @jfgrimm Seems like intel-compilers fail due to a dependency on `fftw3.h`, if I am not mistaken that is an MKL dependency, so I don't think intel-compilers will work.

Reply:
Yes it does dep on fftw3.h so must be at intel level, you need to rename the easyconfig to match the toolchain.

Reply:
That should take care of it.